### DISCUSSION TITLE
Does Avalonia support RTOS?

### DISCUSSION THREAD

**Discussion by xwang-888** [AvaloniaUI/Avalonia#11600] [General]
Does Avalonia support RTOS?

**Comment by kekekeks** [comment]
Do you have some specific RTOS in mind? In general our main dependencies are:
- netstandard2.0-compatible .NET runtime (threading support is optional, filesystem access is optional)
- Skia
- harfbuzz

If you can get those to work with your RTOS, Avalonia would run there, but you'll need to implement a platform backend.
